extensions/htlc/besu-eth/src/main/json/openapi.json

```diff
@@ -0,0 +1,487 @@
+            "type": "string",
+            "nullable": false
+          },
+          "web3SigningCredential": {
+            "description": "Web3SigningCredentialType",
+            "$ref": "../../../../../../packages/cactus-plugin-ledger-connector-besu/src/main/json/openapi.json#/components/schemas/Web3SigningCredential",
+            "nullable": false
+          },
+          "gas": {
+            "oneOf": [
+              {
+                "type": "string"
+              },
+              {
+                "type": "number"
+              }
+            ]
+          }
+        }
+      },
+      "WithdrawReq": {
+        "type": "object",
+        "required": [
+          "id",
+          "secret",
+          "contractAddress",
+          "connectorId",
+          "web3SigningCredential"
+        ],
+        "properties": {
+          "id": {
+            "type": "string",
+            "nullable": false
+          },
+          "secret": {
+            "type": "string",
+            "nullable": false
+          },
+          "contractAddress": {
+            "description": "Contract address",
+            "type": "string",
+            "nullable": false
+          },
+          "connectorId": {
+            "description": "connectorId for the connector besu plugin",
+            "type": "string",
+            "nullable": false
+          },
+          "web3SigningCredential": {
+            "description": "Web3SigningCredentialType",
+            "$ref": "../../../../../../packages/cactus-plugin-ledger-connector-besu/src/main/json/openapi.json#/components/schemas/Web3SigningCredential",
+            "nullable": false
+          },
+          "gas": {
+            "oneOf": [
+              {
+                "type": "string"
+              },
+              {
+                "type": "number"
+              }
+            ]
+          }
+        }
+      },
+      "InitializeReq": {
+        "type": "object",
+        "required": [
+          "web3SigningCredential",
+          "connectorId"
+        ],
+        "properties": {
+          "web3SigningCredential": {
+            "description": "Web3SigningCredentialType",
+            "$ref": "../../../../../../packages/cactus-plugin-ledger-connector-besu/src/main/json/openapi.json#/components/schemas/Web3SigningCredential",
+            "nullable": false
+          },
+          "connectorId": {
+            "type": "string",
+            "nullable": false
+          },
+          "gas": {
+            "type": "number"
+          }
+        }
+      }
+    },
+    "responses": {
+      "GetStatusResp": {
+        "description": "Array ids uint256",
+        "content": {
+          "application/json": {
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "integer",
+                "format": "uint256"
+              }
+            }
+          }
+        }
+      },
+      "GetSingleStatusResp": {
```

Comment:
@AzaharaC Do we still need this/plan to use it later? If not, I recommend to remove.